### PR TITLE
Announce new workshop

### PR DESCRIPTION
**Pull request by vsmart**
From here : https://github.com/clojurebridge-berlin/organization/issues/99 
- Hold until registration and coach form is open

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,28 +12,28 @@
...
-
-      <div class="col-md-9">
-        <div id="signup-form-wrapper">
-          {% include announcement_list_signup_form_slim.html %}
-        </div>
+      <div class="col-xs-12 col-sm-12 col-md-4">
+        <a href="https://docs.google.com/forms/d/1lNwdYa6zl4LVXk7l5jkgDWYtnM7TxHHMqoE08-Vc26Y"><button type="button" class="btn btn-success">ATTEND | TEILNEHMEN</button></a>
+      </div>
+      <div class="col-xs-12 col-sm-12 col-md-4">
+        <a href="TO-DO"><button type="button" class="btn btn-default coach-button">COACH</button></a>
```
Comment (vsmart) [inline index.html]: Todo: Insert Coach form link here

Comment (Realtin) [reply index.html]: Coaches Form  https://goo.gl/forms/K62AAYVYdwDScFmM2

Comment (Realtin) [reply index.html]: attendee Form https://goo.gl/forms/WRwYJR2VP2qoSnFg1

**Comment by vsmart** [comment]
@Realtin gave a 👍. Merging this 🎉

**Pull request merged**